src/main/java/org/z3950/zing/cql/CQLParser.java

```diff
@@ -260,7 +265,18 @@ private CQLNode parseTerm(String index, CQLRelation relation)
 	    debug("index='" + index + ", " +
 		  "relation='" + relation.toCQL() + "'");
 	}
-	CQLTermNode node = new CQLTermNode(index, relation, all.toString());
+        CQLNode node = null;
+        if (this.andMultipleTerms && "=".equals(relstr)) {
```

Comment:
For the "all" relation this is a correct expansion.
For the "==", the "adj" and the "any" relation this is wrong.

http://www.loc.gov/standards/sru/cql/contextSets/theCqlContextSet.html says:
> =
> This is the default relation, and the server can choose any appropriate relation
> For a string term, either 'adj' or '==' as appropriate for the index and term.

Reply:
The standard does not apply here.. The standard does not allow multiple terms after a field..

Such as title=a b
The standard BNF actually only allows title="a b".

So we are in any case describing something that isn't standard. Believe me I was part of the team (Wrote the grammar)